virttest/guest_agent.py

```diff
@@ -569,6 +569,14 @@ def get_time(self):
         return self.cmd(cmd=cmd)
 
     @error_context.context_aware
+    def guest_info(self):
+        """
+        execute guest-info
+        """
+        cmd = "guest-info"
+        return self.cmd(cmd=cmd)
+
+    @error_context.context_aware
     def set_time(self, nanoseconds=None):
```

Comment:
get_ and set_ are a couple.
Don't insert guest_info between them.
And the doc can be improved, refer to shutdown and other old functions.